frontends/ol-components/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import { PlatformEnum } from "api"
+
+export const PLATFORMS = [
```

Comment:
Any reason for this to be an array? Object might be more convenient for lookups, and if we do
```ts
const PLATFORMS: Record<PlatformEnum, PlatformObject> = {

}
```
we'll get TS warnings if we miss platforms.